dlhub_sdk/tests/dlhub_gha_test.py

```diff
@@ -0,0 +1,229 @@
+import os

```

Comment:
This file is mostly a copy of my previous tests. If they are identical, can we merge the files or delete one of them?

Reply:
Ah yes -- I think after I can get everything running smoothly with the test docs I can merge them together or delete one of them